pyproject.toml

```diff
@@ -1,7 +1,7 @@
 
 [tool.poetry]
 name = "sql-mock"
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
**nitpick:**

We don't need to update that one yet. Instead you can add the changes to the [unreleased section in the Changelog.md](https://github.com/DeepLcom/sql-mock/blob/main/CHANGELOG.md?plain=1#L8-L19).

When the stuff is merged, we can create a release (where we would update the version separately)

Reply:
I added some new docs for the release process to the [contribution page](https://github.com/DeepLcom/sql-mock/blob/main/CONTRIBUTION.md#release-process). Let me know whether they make sense :)